pkg/controller/controller_instance.go

```diff
@@ -115,8 +115,8 @@ type backoffEntry struct {
 type instanceOperationBackoff struct {
 	// lock to be used for accessing retry map
 	mutex       sync.RWMutex
-	instances   map[string]backoffEntry
-	rateLimiter workqueue.RateLimiter // used to calculate next retry time
+	instances   map[string]backoffEntry // Key is instance UUID
+	rateLimiter workqueue.RateLimiter   // used to calculate next retry time, Key is instance UUID
```

Comment:
I would say 'kubernetes metadata UID' for these, right?

Reply:
yes, I updated the comment.